torch/_inductor/overrides.py

```diff
@@ -256,13 +304,20 @@ def fuse_unary(gm: torch.fx.GraphModule):
                     len(node.args[0].users) > 1
                 ):  # Output of computation_node is used by other nodes
                     continue
-                conv = modules[node.args[0].target]
+                computation_node = modules[node.args[0].target]
                 unary_node = modules[node.target]
-                eval_mode = all(not n.training for n in [conv, unary_node])
+                eval_mode = all(not n.training for n in [computation_node, unary_node])
                 if not eval_mode:
                     continue
-                fused_conv = fuse_func(conv, unary_node)
-                replace_node_module(node.args[0], modules, fused_conv)
+
+                # only fuse for linear when the dtype is bf16
+                if bf16_only_node(computation_node) and not is_bfloat16_module(
```

Comment:
simpler to just do `type(computation_node) in [nn.Linear]` than factoring out a function `bf16_only_node` here?

Reply:
yes, changed.